bftengine/tests/simpleKVBC/src/ReplicaImp.h

```diff
@@ -33,19 +43,20 @@ namespace SimpleKVBC {
 	{
 	public:
 		// IReplica methods
-																	 
+                
 		virtual Status start() override;
 		virtual Status stop() override;
 		virtual RepStatus getReplicaStatus() const override;
 		virtual bool isRunning() const override;
-																				 
+                virtual void monitor() const override {}

```

Comment:
nit: alignment … oh, it's a tabs-vs-spaces issue. Not a blocker.

Reply:
I formatted the file.